src/internal/k8s/state.go

```diff
@@ -35,14 +35,15 @@ func LoadZarfState() types.ZarfState {
 		_ = json.Unmarshal(match.Data[ZarfStateDataKey], &state)
 	}
 
-	message.Debug(state)
+	message.Debug(message.JsonValue(state))
```

Comment:
Probably should make this a `message.Debugf( ... )` so that we can describe what is getting logged. It would probably take me a little longer to realize that the random json is the zarf state